libraries/ui/src/default-config/tailwind.css

```
@@ -113,6 +113,34 @@
     @apply w-full max-w-max-width mx-auto px-spacing-x pt-6 pb-10 border-b border-color-divider overflow-hidden flex flex-col;
 }
 
+@utility nav-link-animation {
+    @apply relative transition-[translate] duration-200;
+
+    &:hover {
+        @apply text-current -translate-y-[4px];
+    }
+
+    &::after {
+        content: "";
+        @apply invisible absolute bg-bluedot-normal transition-[translate] duration-200 left-0 bottom-0 w-full h-[2px];
+    }
```

we can combine all of this into a single block with tailwind:

```suggestion
@utility nav-link-animation {
    @apply relative transition-[translate] duration-200 after:content-[""] after:invisible after:absolute after:bg-bluedot-normal after:transition-[translate] after:duration-200 after:left-0 after:bottom-0 after:w-full after:h-[2px] hover:text-current hover:-translate-y-[4px] after:hover:visible after:hover:translate-y-[4px];
```

---

Nice, I just split it onto a few lines to make it more readable:
```css
@utility nav-link-animation {
    @apply relative transition-[translate] duration-200;
    @apply hover:text-current hover:-translate-y-[4px];
    @apply after:content-[""] after:invisible after:absolute after:bg-bluedot-normal after:transition-[translate] after:duration-200 after:left-0 after:bottom-0 after:w-full after:h-[2px];
    @apply hover:after:visible hover:after:translate-y-[4px];
}
```